src/Bijectors.jl

```diff
@@ -190,6 +192,44 @@ function link(
     return y
 end
 
+function link_jacobian(
+    d::SimplexDistribution, 
+    x::AbstractVector{T}, 
+    ::Type{Val{proj}} = Val{true}
+) where {T<:Real, proj}
+    y, K = similar(x), length(x)
+    dydxt = similar(x, length(x), length(x))
+    @inbounds dydxt .= 0
+    ϵ = _eps(T)
+    sum_tmp = zero(T)
+
+    @inbounds z = x[1] * (one(T) - 2ϵ) + ϵ # z ∈ [ϵ, 1-ϵ]
+    @inbounds y[1] = StatsFuns.logit(z) + log(T(K - 1))
+    @inbounds dydxt[1,1] = (1/z + 1/(1-z)) * (one(T) - 2ϵ)
+    @inbounds @simd for k in 2:(K - 1)
+        sum_tmp += x[k - 1]
+        # z ∈ [ϵ, 1-ϵ]
+        # x[k] = 0 && sum_tmp = 1 -> z ≈ 1
+        z = (x[k] + ϵ)*(one(T) - 2ϵ)/((one(T) + ϵ) - sum_tmp)
+        y[k] = StatsFuns.logit(z) + log(T(K - k))
+	    dydxt[k,k] = (1/z + 1/(1-z)) * (one(T) - 2ϵ)/((one(T) + ϵ) - sum_tmp)
+	    for i in 1:k-1
+		    dydxt[i,k] = (1/z + 1/(1-z)) * (x[k] + ϵ)*(one(T) - 2ϵ)/((one(T) + ϵ) - sum_tmp)^2
+		end
+    end
+    @inbounds sum_tmp += x[K - 1]
+    @inbounds if proj
+        y[K] = zero(T)
+    else
+        y[K] = one(T) - sum_tmp - x[K]
+	    @simd for i in 1:K
+		    dydxt[i,K] = -1
+		end        
+    end
+
+    return UpperTriangular(dydxt)'
```

Comment:
Should the adjoint be taken _before_ instantiating as `UpperTriangular`, i.e. `UpperTriangular(dydxt')`?

Reply:
`dydxt` is upper triangular, and it is the transpose of the Jacobian. So we should either do `UpperTriangular` then adjoint, or adjoint then `LowerTriangular`. It doesn't make a difference in the underlying data structure because both are lazy wrappers, only dispatch will be different. I will leave it as is.